idaes/generic_models/unit_models/column_models/column.py

```diff
@@ -0,0 +1,1144 @@
+                              self.liquid_phase.length_domain,
+                              initialize=0.001,
+                              doc='Volumetric liquid holdup [-]')
+
+        def rule_holdup_vap(blk, t, x):
+            return blk.eps_ref - blk.holdup_liq[t, x]
+
+        self.holdup_vap = Expression(self.flowsheet().time,
+                                     self.vapor_phase.length_domain,
+                                     rule=rule_holdup_vap,
+                                     doc='Volumetric vapor holdup [-]')
+        
+        # Calculating gas velocity at flooding point
+        def rule_flood_velocity(blk, t, x):
+            x_liq = blk.liquid_phase.length_domain.at(blk.zi[x].value)
+            dens_liq = blk.liquid_phase.properties[t, x_liq].mw/\
+                    blk.liquid_phase.properties[t, x_liq].vol_mol_phase['Liq']
+                    
+            dens_vap = blk.vapor_phase.properties[t, x].mw/\
+                    blk.vapor_phase.properties[t, x].vol_mol_phase['Vap']
+                    
+            H = (blk.liquid_phase.properties[t, x_liq].flow_mass_phase['Liq']/\
+                blk.vapor_phase.properties[t, x].flow_mass_phase['Vap'])*\
+                (dens_liq/dens_vap)**0.5
+                
+            mu_water = 0.001*pyunits.Pa*pyunits.s
+            
+            g_ref = CONST.acceleration_gravity
+            gref = pyunits.convert(g_ref, to_units=pyunits.ft/(pyunits.s)**2)
+            
+            aref = pyunits.convert(blk.packing_specific_area, to_units=((pyunits.ft)**2)/(pyunits.ft)**3)
+            
+            return ((gref*((blk.eps_ref)**2)/aref)*(dens_liq/dens_vap)*\
+                ((blk.liquid_phase.properties[t, x_liq].visc_d_phase['Liq']/mu_water)*(-0.2))*\
+                    exp(-4*(H)**(0.25)))**0.5
+                
+        self.gas_velocity_flood = Expression(self.flowsheet().time,
+                                self.vapor_phase.length_domain,
+                                rule=rule_flood_velocity,
+                                doc='Gas velocity at flooding point')
+            
+        # Flooding fraction 
+        def rule_flood_fraction(blk, t, x):
+            return blk.velocity_vap[t, x]/blk.gas_velocity_flood[t, x]
+            
+        self.flood_fraction = Expression(self.flowsheet().time,
+                                    self.vapor_phase.length_domain,
+                                    rule=rule_flood_fraction,
+                                    doc='Flooding fraction (expected to be below 0.8)')
+        
+        # Mass and heat transfer terms
+        
+        # Mass transfer terms
+        self.pressure_equil = Var(
+            self.flowsheet().time,
+            self.vapor_phase.length_domain,
+            equilibrium_comp,
+            domain=NonNegativeReals,
+            initialize=500,
+            units=pyunits.Pa,
+            doc='Equilibruim pressure of diffusing components at interface')
```

Comment:
I would remove "diffusing" from this comment, as the generic model does not concern itself with diffusion (that is for the more detailed/specific models to worry about).